localization/gyro_odometer/README.md

```diff
@@ -28,4 +27,13 @@
 
 ## Assumptions / Known limits
 
-TBD.
+- [Assumption] The frame_id of input twist message must be base_link
+
+- [Assumption] The covariances in the input messages must be properly entered
+
+- [Assumption] If both velocity and angular velocity are sufficiently small, the angular velocity is fixed at zero
+- This is because there was a bias in the angular velocity of the IMU, which sometimes caused a estimation that the vehicle was rotating even though it is actually stopped.
+
+- [Limitation] The cycle of the output messages depend on the cycle of the IMU message
+
+- [Limitation] The variances of linear.y and linear.z are output message is assigned large values
```

Comment:
Not sure what this means... Do you mean linear.y and linear.z in the output message have large values?

Reply:
Thank you!
I'll remove pitch and roll from the text, because this PR(https://github.com/autowarefoundation/autoware.universe/pull/407) will be coming in soon.